demo/App.vue

```diff
@@ -54,10 +54,10 @@ v-app.app
           girder-data-browser(
               ref="girderBrowser",
               :location.sync="location",
-              :select-enabled="selectEnabled",
+              :selection="selectEnabled",
```

Comment:
`selection` is a little ambiguous, easy to confuse with `selected`.  `selectable` would be preferred to match `draggable`, otherwise keep `select-enabled`.

Reply:
`selectable` does sound better, I'll update it.